docs/sdks/overview.mdx

```diff
@@ -31,3 +31,36 @@ From local development to production, Infisical SDKs provide the easiest way for
         Manage secrets for your PHP application on demand
     </Card>
 </CardGroup>
+
+## FAQ
+
+<AccordionGroup>
+    <Accordion title="Isn't it inefficient if my app makes a request every time it needs a secret?">
+        The client SDK caches every secret and implements a 5-minute waiting period before
+        re-requesting it. The waiting period can be controlled by setting the `cacheTTL` parameter at
+        the time of initializing the client.
+        
+        Note: The exact parameter name may differ depending on the language.
+    </Accordion>
+       <Accordion title="Can I attach the environment variables to my process environment?">
+        Yes you can! The client SDK provides a method to attach the secrets to your process environment.
+        When using the `listSecrets()` method, you can pass a `attachToProcessEnv` parameter, which tells the SDK to attach all the found secrets to your process environment.
+        
+        Note: The exact parameter name may differ depending on the language.
+    </Accordion>
+    <Accordion title="What if a request for a secret fails?">
+        The SDK caches every secret and falls back to the cached value if a request fails. If no cached value is found, and the request fails, then the SDK throws an error.
+    </Accordion>
+    <Accordion title="What's the point if I still have to manage a token for the SDK?">
```

Comment:
The new authentication method significantly lowers the risk of token exposure. Here's how it works:

- Limited-use client secrets: You can create client IDs and secrets that can only be used a certain number of times (e.g., once) or expire after a short time (e.g., 20 seconds).
- In-memory access tokens: The access token generated from these limited-use secrets is stored securely in memory, not saved or transmitted elsewhere.
- Expired client secrets: After the token is created, the original client secret becomes invalid, preventing any further use.

Also client secrets will typically be delivered via trusted brokers method ensuring client id and client secret only comes together at just-in-time-use.

Let's remove this question from the FAQ since this has a more nuanced answer now compared to when we had just plain service tokens